types/matrix-appservice-bridge/index.d.ts

```diff
@@ -0,0 +1,438 @@
+// Type definitions for matrix-appservice-bridge 1.11
+// Project: https://github.com/matrix-org/matrix-appservice-bridge
+// Definitions by: Huan LI (李卓桓) <https://github.com/huan>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.1
+
+/// <reference types="node" />
+
+import { EventEmitter } from 'events';
+
+import {
+    EventType,
+    MsgType,
+    MatrixClient,
+    MembershipType,
+} from 'matrix-js-sdk';
+
+export type Controller = any;
+
+// FIXME: declare it in the right way
+export class AppServiceRegistration {
+    static generateToken(): string;
+
+    // FIXME: ERROR: 21:8    no-unnecessary-class          Every member of this class is static. Use namespaces or plain objects instead.
```

Comment:
There are multiple fixme comments here, Please fix those

Reply:
Thanks for reminding this. 

Fixed.